Codeinwp/visualizer
# fix: show charts based on the selected library

### Summary


- Pie & Line ChartJS are now available for Free users. The Library Select has been enabled to allow the change.
- Fixed a minor typo in the CSS class name.
- The Library Select filters the available charts when the user changes its values directly. (By default, all types of charts are shown to showcase all available features ) 
- When you hover over the chart options, `cursor: pointer` will be used to improve the UX (suggested by @mghenciu)

> [!IMPORTANT]
> The Library Select design will change when https://github.com/Codeinwp/visualizer/pull/1127 will be merged

> [!NOTE]
> To preserve the CSS positions, I duplicated the original image and changed only the available chart for each library.


### Will affect visual aspect of the product

YES

### Screenshots 

#### Free

https://github.com/Codeinwp/visualizer/assets/17597852/c11aa5e1-7b4a-4e3d-8f3a-022f94295e52

#### Pro
https://github.com/Codeinwp/visualizer/assets/17597852/63d4f500-8699-4f74-8bc6-f6af4d74b0b6

### Test instructions


- Install Visualizer Free and Pro
- Create a new chart
- When you select the library, the chart list should change and show only available charts for that library.

## Check before Pull Request is ready:



Closes https://github.com/Codeinwp/visualizer-pro/issues/456 & https://github.com/Codeinwp/visualizer-pro/issues/464
